### ISSUE TITLE
Elasticsearch 7.2.0 fails to install

### ISSUE THREAD

**Issue by adig14** [elastic/windows-installers#315]
I am trying to install elasticsearch 7.2.0 on Windows 10 Pro using the "elasticsearch-7.2.0.msi" from a command prompt running as Administrator. I have enclosed the install log file

[MSI25f20.LOG](https://github.com/elastic/windows-installers/files/3354018/MSI25f20.LOG)
 

Previous I had a successful implementation of v6.6 but this stopped working after I installed the Phonetic plugin so I uninstalled this via Add/Remove Programs and then initially encountered some issues with the msi complaining about ES_HOME being set but this was not visible in Environment variables but was visible from command prompt by doing "echo %ES_HOME%"

This was then removed by typing "set ES_HOME=" and the msi would run but once completed finished with "An error occurred during installation!" and produce the enclosed log file.

The installation is being run from an administrator account from a command prompt running as administrator. The java version is....

java version "1.8.0_211"
Java(TM) SE Runtime Environment (build 1.8.0_211-b12)
Java HotSpot(TM) 64-Bit Server VM (build 25.211-b12, mixed mode)

**Comment by UkrZilla** [comment]
Investigated problem and it seems the issue in this jvm.options line:

`9-:-Xlog:gc*,gc+age=trace,safepoint:file=logs/gc.log:utctime,pid,tags:filecount=32,filesize=64m`

also in event manager I see:

```
Elastic.ProcessHosts.Process.StartupException: Could not evaluate jvm.options file: D:\Elasticsearch\Config\jvm.options result: Exception in thread "main" java.lang.RuntimeException: starting java failed with [1]
output:
[0.002s][error][logging] Error opening log file 'logs/gc.log': No such file or directory
[0.002s][error][logging] Initialization of output 'file=logs/gc.log' using options 'filecount=32,filesize=64m' failed.
error:
OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release.
Invalid -Xlog option '-Xlog:gc*,gc+age=trace,safepoint:file=logs/gc.log:utctime,pid,tags:filecount=32,filesize=64m', see error log for details.
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
	at org.elasticsearch.tools.launchers.JvmErgonomics.flagsFinal(JvmErgonomics.java:111)
	at org.elasticsearch.tools.launchers.JvmErgonomics.finalJvmOptions(JvmErgonomics.java:79)
	at or...
```

If I choose install options: "Do no start service after Installer done" I can comment this line and service successfully started :)
I guess it is related to custom Logs dir

@russcam Can you please fix it?

Thanks

**Comment by macko1977** [comment]
Got the very same problem. Installer fails. Log file attached. 

Have tried to install as a service as well as on-demand start. Both fail.
[MSI4817.txt](https://github.com/elastic/windows-installers/files/3364723/MSI4817.txt)

**Comment by carlos-mora** [comment]
Reviewing @macko1977 report looks like mine, showing problems in the plugins install section. Mine failed in spell addon.
[MSI509e0.LOG](https://github.com/elastic/windows-installers/files/3381389/MSI509e0.LOG)

**Comment by raschmi** [comment]
[MSI7379b.LOG](https://github.com/elastic/windows-installers/files/3383904/MSI7379b.LOG)

**Comment by Phrozyn** [comment]
[MSId3008.log](https://github.com/elastic/windows-installers/files/3393172/MSId3008.log)

MSI Failure on plugin installations.

**Comment by russcam** [comment]
install errors related to 

```
Error opening log file 'logs/gc.log': No such file or directory
```

have been addressed in #316 and will be fixed in the next release.

**Comment by russcam** [comment]
Closing this issue as has been fixed in 7.2.1 and 7.3.0

**Comment by pelmered** [comment]
Is this fixed in version 7.5.1?
I looks like I'm getting the same error.

I get this error:
```
Exception in thread "main" java.lang.RuntimeException: starting java failed with [1]
output:
[0.003s][error][logging] Error opening log file 'logs/gc.log': No such file or directory
[0.003s][error][logging] Initialization of output 'file=logs/gc.log' using options 'filecount=32,filesize=64m' failed.
error:
OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release.
Invalid -Xlog option '-Xlog:gc*,gc+age=trace,safepoint:file=logs/gc.log:utctime,pid,tags:filecount=32,filesize=64m', see error log for details.
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
	at org.elasticsearch.tools.launchers.JvmErgonomics.flagsFinal(JvmErgonomics.java:118)
	at org.elasticsearch.tools.launchers.JvmErgonomics.finalJvmOptions(JvmErgonomics.java:86)
	at org.elasticsearch.tools.launchers.JvmErgonomics.choose(JvmErgonomics.java:59)
	at org.elasticsearch.tools.launchers.JvmOptionsParser.main(JvmOptionsParser.java:92)
```

I'm running macOS Catalina and installed elasticsearch with homebrew (`elastic/tap/elasticsearch-full` from this tap: https://github.com/elastic/homebrew-tap).
